source/code/scxsystemlib/disk/staticdiskpartitionenumeration.cpp

```diff
@@ -993,15 +993,14 @@ namespace SCXSystemLib
         
         try
         {
-            std::wstring command = parted_path + L" -l";
+            std::wstring command = parted_path + L" -ls";
             SCX_LOGTRACE(m_log, L"Invoking command : \"" + command + L"\"");
             int ret = m_deps->Run(command, processInput, processOutput, processErr, 15000);
 
-            std::string errOut = processErr.str();
```

Comment:
I think there was a prior bug here. Shouldn't processErr.str() have been dumped in trace as well?